views/clickhouse.go

```diff
@@ -75,13 +86,38 @@ func (c *ClickhouseClient) QueryRealtimeViewsEvents(ctx context.Context, spec Qu
 }
 
 func buildRealtimeViewsEventsQuery(spec QuerySpec) (string, []interface{}, error) {
-	var query squirrel.SelectBuilder
-	if spec.From == nil && spec.To == nil {
-		query = currentEventsQuery(spec)
-	} else {
-		query = timeRangeEventsQuery(spec)
+	query := squirrel.Select(
+		"count(distinct session_id) as view_count").
```

Comment:
I think we only really wanted to match outputs in case we were keeping it on the same API. If there's no customer need for this info, might be easier to not include it for now.

Reply:
Yes, if we have separate APIs, then I think we don't need to match the output. So IMO we should include what makes sense for this particular endpoint, in other words:
- Include `error_rate`
- Do not include `buffer_ratio`

Reply:
Btw. @ecmulli could you update the query in the Notion doc?
1. We don't have `errors` column in `viewership_events`.
2. Shouldn't we also filter the events everywhere like `where event_type='heartbeat'`? Otherwise, if there is an error in the browser and it sends a couple of `error` events, then we will count them many times I guess.